src/utils.js

```diff
@@ -122,3 +122,17 @@ export function filterCsvString(str, bbox4326) {
   const filteredString = lines.join('\n');
   return filteredString;
 }
+
+/**
+ * Check is string can be parsed to object
+ * @param string
+ * @returns {boolean}
+ */
+export function isStringJson(string) {
```

Comment:
-> `isSerializedJson`?

Reply:
I updated this function. Now, it returns parsed json or undefined to skip the second parse